<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="5" total="6" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-03-01T17:34:54 EST" name="Default suite" finished-at="2023-03-01T17:34:55 EST" duration-ms="226">
    <groups>
    </groups>
    <test started-at="2023-03-01T17:34:54 EST" name="Default test" finished-at="2023-03-01T17:34:55 EST" duration-ms="226">
      <class name="com.RestAssured.PostRequest">
        <test-method signature="postTest6()[pri:0, instance:com.RestAssured.PostRequest@29176cc1]" started-at="2023-03-01T17:34:54 EST" name="postTest6" finished-at="2023-03-01T17:34:55 EST" duration-ms="144" status="FAIL">
          <exception class="org.json.JSONException">
            <message>
              <![CDATA[Duplicate key "fname"]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.json.JSONException: Duplicate key "fname"
at org.json.JSONObject.putOnce(JSONObject.java:1894)
at com.RestAssured.PostRequest.postTest6(PostRequest.java:203)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.json.JSONException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postTest6 -->
      </class> <!-- com.RestAssured.PostRequest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
